XRTK-Core/Packages/com.xrtk.core/Definitions/Controllers/UnityInput/GenericJoystickController.cs

```diff
@@ -101,7 +101,8 @@ protected void UpdateButtonData(MixedRealityInteractionMapping interactionMappin
             {
                 case DeviceInputType.TriggerPress:
                     Debug.Assert(!string.IsNullOrEmpty(interactionMapping.AxisCodeX), $"[{interactionMapping.Description}] Axis mapping does not have an Axis defined");
-                    interactionMapping.BoolData = Input.GetAxisRaw(interactionMapping.AxisCodeX).Equals(interactionMapping.InvertXAxis ? -1f : 1f);
+                    var processor = interactionMapping.GetInputProcessors<InvertSingleAxisProcessor>();
+                    interactionMapping.BoolData = Input.GetAxisRaw(interactionMapping.AxisCodeX).Equals(processor.Invert ? -1f : 1f);
```

Comment:
I'd like to revisit this to see if there's a better way to evaluate and set the bool data without having to know about the processor